inference/environments/minimal-ubuntu20.04-py38-cpu-inference/context/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/azureml/inference-base-2004:{{latest-image-tag}}
+
+RUN conda create -n minimal \
+    python=3.8.13 pip=22.1.2
```

Comment:
Also, you're not referencing your conda_dependencies.yaml file.

Reply:
I will remove this.